operator/tests/e2e/e2e_test.go

```diff
@@ -137,16 +139,16 @@ func TestMain(m *testing.M) {
 
 	log, err := newLogger("TestMain")
 	if err != nil {
-		fmt.Printf("error creating logger: %v", err)
+		fmt.Print(fmt.Errorf("error creating logger: %w", err))
```

Comment:
Nit: I assume you did this for the linter, but it seems like the original version was fine